### PR TITLE
Add PIPP version 2.5.9

### PR DESCRIPTION
**Pull request by MCOfficer**

### CODE REVIEW

File: bucket/pipp.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        "64bit": {
+            "url": "https://sites.google.com/site/astropipp/downloads/pipp_install_x64_2.5.9.zip",
+            "hash": "231f08bf36aa812946158d6239861bd9a3a328d33124afd357a35dc28d623efe"
+        },
+        "32bit": {
+            "url": "https://sites.google.com/site/astropipp/downloads/pipp_install_x86_2.5.9.zip",
+            "hash": "87dfaa846e75e265a004961aa785b5544ec7cf8ae0cf4c7443a9c677172bba0d"
+        }
+    },
+    "innosetup": true,
```
Comment (MCOfficer) [inline bucket/pipp.json]: As i just noticed, now that innosetup is fixed, [this is bullshit](https://github.com/lukesampson/scoop/issues/3339#issuecomment-487362746)

Which poses the question: can i extract the archive and *then* run innosetup?
and can i run innosetup on a file with version-specific name? autoupdate doesn't seem to set any values for `installer`.

Comment (Ash258) [reply bucket/pipp.json]: It is not innosetup at all.

A

**Comment by Ash258** [comment]
Fixed manifest:

```json
{
    "description": "An Application designed for pre-processing planetary images before stacking them with image stacking software such as Registax.",
    "homepage": "https://sites.google.com/site/astropipp/home",
    "version": "2.5.9",
    "license": {
        "identifier": "Freeware",
        "url": "https://sites.google.com/site/astropipp/home#TOC-Acknowledgements"
    },
    "architecture": {
        "64bit": {
            "url": "https://sites.google.com/site/astropipp/downloads/pipp_install_x64_2.5.9.zip",
            "hash": "231f08bf36aa812946158d6239861bd9a3a328d33124afd357a35dc28d623efe"
        },
        "32bit": {
            "url": "https://sites.google.com/site/astropipp/downloads/pipp_install_x86_2.5.9.zip",
            "hash": "87dfaa846e75e265a004961aa785b5544ec7cf8ae0cf4c7443a9c677172bba0d"
        }
    },
    "installer": {
        "script": [
            "(Get-ChildItem \"$dir\" '*.exe').FullName | Expand-7zipArchive -DestinationPath \"$dir\" -Removal",
            "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\uninst.exe\" -Recurse"
        ]
    },
    "bin": "PIPP.exe",
    "shortcuts": [
        [
            "PIPP.exe",
            "PIPP"
        ]
    ],
    "checkver": {
        "url": "https://sites.google.com/site/astropipp/downloads",
        "regex": "Windows Installer \\(v([\\d\\.]+)\\)"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://sites.google.com/site/astropipp/downloads/pipp_install_x64_$version.zip"
            },
            "32bit": {
                "url": "https://sites.google.com/site/astropipp/downloads/pipp_install_x86_$version.zip"
            }
        }
    }
}
```

**Comment by MCOfficer** [comment]
You're godsent, I was still meddling around with the installer.

I still think it's Inno-based (at least that's what `strings` tells me), but you're right, in this case one can just extract it.
I'm divided about the license url, though - acknowledgements are fine and all, but it's not the license the installer gives you, in fact it's not a license at all. But i'll leave that question to you maintainers.

**Comment by r15ch13** [comment]
> I still think it's Inno-based (at least that's what `strings` tells me),

image
The Logo and string say it's an NSIS based installer 😕

**Comment by MCOfficer** [comment]
i concede - that logo might've given it away, but not the footer... thanks dark theme 😄 
grafik

**Comment by r15ch13** [comment]
`utvideo-16.1.0-win.exe`
image

**Pull request merged**